forward_velocity:
  slope: -3.0
  weight: 3.0
lateral_velocity:
  slope: -3.0
  weight: 2.0
angular_velocity:
  slope: -1.5
  weight: 1.5
balance:
  vz_slope: -2.5
  wxy_slope: -2.0
  weight: 1.0
twist:
  weight: -0.6
foot_rhythm:
  weight: 1.0
  c1: 1.0
  height_thresh: 0.011
  offset: 0.007
joint_constraints:
  weight: -0.8
rg_phase:
  weight: 1.0